mirrormanager2/lib/hostconfig.py

```diff
@@ -25,9 +25,14 @@
 
 import mirrormanager2.lib
 
+import logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
Sure, I just used this since it's the only thing going into the httpd logs, so we can easily find which part of the code wrote this log.